vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/index/IndexDeleteEvaluatorFactory.java

```diff
@@ -45,8 +45,8 @@ public IndexDeleteEvaluatorFactory(IScalarEvaluatorFactory[] args) {
         super(args);
     }
 
-    @Override
-    protected IScalarEvaluator createEvaluator(IHyracksTaskContext ctx, IScalarEvaluator[] args) throws AlgebricksException {
+    @Override protected IScalarEvaluator createEvaluator(IHyracksTaskContext ctx, IScalarEvaluator[] args)
```

Comment:
I am not sure why this changed. Our formatting should keep @Override on a separate line.